blockly/core/field_variable.js

```diff
@@ -124,7 +124,7 @@ Blockly.FieldVariable.prototype.setValue = function(newValue) {
  * @return {!Array.<string>} Array of variable names.
  * @this {!Blockly.FieldVariable}
  */
-Blockly.FieldVariable.dropdownCreate = function() {
```

Comment:
This is a good catch, which is still present in blockly master. If you'd like, you could submit this fix upstream as well.